MatrixKit/Utils/EventFormatter/MXKEventFormatter.m

```diff
@@ -997,11 +997,11 @@ - (NSAttributedString *)attributedStringFromEvent:(MXEvent *)event withRoomState
             {
                 if (isRoomDirect)
                 {
-                    displayText = [MatrixKitL10n noticeRoomAliasesForDm:aliases];
+                    displayText = [MatrixKitL10n noticeRoomAliasesForDm:aliases.description];
```

Comment:
I wonder if it would be better to use `[aliases componentsJoinedByString:@", "] to have a more predictable behavior?
Do you have an example of the rendering?

Reply:
FWIW, there's a new `NSListFormatter` for iOS 13+ that provides locale aware lists that might be worth looking at too.